eve/utils.py

```diff
@@ -386,8 +386,27 @@ def validate_filters(where, resource):
 
     def validate_filter(filter):
         for key, value in filter.items():
-            if '*' not in allowed and key not in allowed:
-                return "filter on '%s' not allowed" % key
+            if '*' not in allowed:
+                def recursive_check_allowed(filter_key, allowed_filters):
+                    # Filter key can be a plain key (e.g. "foo") or a dotted
```

Comment:
I would rather move this comment to the documentation.

Reply:
Hi @nicolaiarocci , thanks for your review!
Just to be sure, did you mean ALLOWED_FILTERS? Also, do you want me to remove this comment from the code once it will be present in the documentation?

Reply:
Yes, remove it from the code. And yes that's what I meant.

Reply:
Ok, I updated the PR with the requested changes ;)